### DISCUSSION TITLE
IPv4 and IPv6 at the same time

### DISCUSSION THREAD

**Discussion by KopiNinja** [bluenviron/mediamtx#2905] [Questions]
### Question

How can i use IPv4 and IPv6 at the same time? Is this possible?

If i leave `webrtcIPsFromInterfaces` enabled and put an IPv6 on `webrtcAdditionalHosts`, i get:
`[WebRTC] [session fb2c0a86] closed: deadline exceeded while waiting connection`

If i disable `webrtcIPsFromInterfaces` and put an IPv4 and an IPv6 in `webrtcAdditionalHosts`, it doesn't work either and i get a lot of `[WebRTC] [session 0444d1fd] 16 RTP packets lost` when i start streaming with OBS.

I'm using mediamtx v1.4.2 and OBS 30.0.2 on Windows 10.

**Comment by aler9** [comment]
moved in #3227
